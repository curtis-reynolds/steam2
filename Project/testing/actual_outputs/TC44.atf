06_nonadmin________FS_000100.00
00_admin___________AA_000000.00
