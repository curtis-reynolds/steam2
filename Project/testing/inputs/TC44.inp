login
admin
addcredit
nonadmin
100
