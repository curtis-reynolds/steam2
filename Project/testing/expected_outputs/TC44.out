Login successful.
Credit added successfully. New credit for user nonadmin: 100
